checkers/python/jwt-python-none-alg.yml

```diff
@@ -0,0 +1,42 @@
+language: py
+name: jwt-python-none-alg
+message: Detected the usage of `none` in the algorithms parameter in JWT token
```

Comment:
Can you refine the message here? Currently, it's describing what we have detected (which is an implementation detail). We should instead tell people what's wrong with their code on an abstract level.